### ISSUE TITLE
'formatTime is not a function' when rendering FormattedTime

### ISSUE THREAD

**Issue by vasa-chi** [yahoo/react-intl#165]
`formatTime` isn't on context, thus `const {formatTime} = this.context` gives undefined. Here is a small patch to fix this error:

```patch
index b9549b9..45500a8 100644
--- a/src/components/time.js
+++ b/src/components/time.js
@@ -14,7 +14,7 @@ export default class FormattedTime extends Component {
     }
 
     render() {
-        const {formatTime} = this.context;
+        const {formatTime} = this.context.intl;
         const props        = this.props;
 
         let formattedTime = formatTime(props.value, props);
```

**Comment by ericf** [comment]
Thanks, I'll fix this.

**Comment by ooflorent** [comment]
Is there any release date for this fix?

**Comment by ericf** [comment]
> Is there any release date for this fix?

I'll do another preview release this week, I'm working on addressing other things as well…

**Comment by ericf** [comment]
@vasa-chi @ooflorent this was released yesterday in [v2.0.0-pr-2](https://github.com/yahoo/react-intl/releases/tag/v2.0.0-pr-2)
